### PR TITLE
Feature/post usability evaluation

### PR DESCRIPTION
**Pull request by BjoernKrollGba**
@alexanderkiel Hi Alex, this PR consists of the changes due to the usibability evaluation and changes requested by Verena Huth. If possible I would like to have the code deployed on test.bbmri.de so that Michael Hummel and others can evaluate changes like the removed splitting of "Donor/Samples" on the top level UI. Thanks!.

### CODE REVIEW

File: src/app/component/search-builder/search-builder.component.scss [hunk 1]
```diff
@@ -22,7 +22,27 @@ h3 {
...
 
 .add-field-wrapper {
-  margin: 0 .5em;
+  margin: 0 .2em;
+}
+
+:host ::ng-deep .add-field,
+:host ::ng-deep .add-field .ui-dropdown-label,
+:host ::ng-deep .add-field .ui-dropdown-trigger {
+  -webkit-transition: all .2s ease-out !important;
```
Comment (alexanderkiel) [inline src/app/component/search-builder/search-builder.component.scss:31]: Is the "!important" really necessary? I would only use it if we see no other way.

File: src/app/component/search-builder/search-builder.component.scss [hunk 2]
```diff
@@ -22,7 +22,27 @@ h3 {
...
+  -o-transition: all .2s ease-out !important;
+  -ms-transition: all .2s ease-out !important;
+  transition: all .2s ease-out !important;
+}
+
+:host ::ng-deep .add-field:hover,
+:host ::ng-deep .add-field:hover .ui-dropdown-trigger,
+:host ::ng-deep .add-field:hover .ui-dropdown-label {
+  -webkit-transition: all .2s ease-in !important;
+  -moz-transition: all .2s ease-in !important;;
```
Comment (alexanderkiel) [inline src/app/component/search-builder/search-builder.component.scss:42]: double semicolons

File: src/styles.scss [hunk 3]
```diff
@@ -21,15 +21,19 @@ body {
...
   -moz-transition: all .2s ease-out;
   -o-transition: all .2s ease-out;
   -ms-transition: all .2s ease-out;
   transition: all .2s ease-out;
 }
 
-*:hover {
+.bbmri-transition:hover,
+.bbmri-transition a:hover,
+.bbmri-transition div:hover {
```
Comment (alexanderkiel) [inline src/styles.scss:36]: You missed the standard and the "-ms" variant here.

File: src/app/component/search-builder/search-builder.component.scss (hunk 1 above)
Comment (alexanderkiel) [reply src/app/component/search-builder/search-builder.component.scss:31]: Why do you not use "bbmri-transition" on the elements? It does the same.

File: src/app/page/search/search.component.ts [hunk 4]
```diff
@@ -17,6 +17,18 @@ import {SampleLocatorConstants} from '../../SampleLocatorConstants';
 })
 export class SearchComponent implements OnInit {
 
+  private static MINIMAL_NUMBER_VALUES_TO_TOP_SHOW_ACTION_BUTTONS = 7;
```
Comment (alexanderkiel) [inline src/app/page/search/search.component.ts:20]: This constant is already defined at another place.

**Review by alexanderkiel — CHANGES_REQUESTED** [review]

File: package.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "sample-locator",
-  "version": "1.3.0",
+  "version": "2.0.0",
```
[the remark this answers is not in the document]
Comment (BjoernKrollGba) [reply package.json:3]: Not really breaking changes. However, it is the first version which does not state that the SL is work in progress.

File: src/app/component/search-builder/search-builder.component.scss (hunk 1 above)
Comment (BjoernKrollGba) [reply src/app/component/search-builder/search-builder.component.scss:31]: I must overwrite the behaviour of the PimeNG framework (dropdown). That's why I must use !important AND cannont specify the behaviour by the CSS-class "bbmri-transition".

File: src/styles.scss (hunk 3 above)
Comment (BjoernKrollGba) [reply src/styles.scss:36]: I do not see where I missed them? My code (lines 34 - 42) looks like:

.bbmri-transition:hover,
.bbmri-transition a:hover,
.bbmri-transition div:hover {
  -webkit-transition: all .2s ease-in;
  -moz-transition: all .2s ease-in;
  -o-transition: all .2s ease-in;
  -ms-transition: all .2s ease-in;
  transition: all .2s ease-in;
}

Comment (alexanderkiel) [reply src/styles.scss:36]: ok was a problem with code folding.

**Review by alexanderkiel — APPROVED** [review]

**Pull request merged**